app/server/models/utils.py

```diff
@@ -48,19 +48,22 @@ def no_expire():
     yield
     s.expire_on_commit = True
 
-
 @event.listens_for(AppQuery, "before_compile", retval=True)
 def filter_by_org(query):
     """A query compilation rule that will add limiting criteria for every
     subclass of OrgBase"""
+    org_check = query._execution_options.get("org_check", False)
     show_deleted = getattr(g, "show_deleted", False) or query._execution_options.get("show_deleted", False)
     show_all = getattr(g, "show_all", False) or query._execution_options.get("show_all", False)
     # We want to support multiple active organizations, but only for select GET requets.
     # This is done through a multi_org flag, very similar to the show_all flag
     multi_org = getattr(g, "multi_org", False) or query._execution_options.get("multi_org", False)
     if show_all and show_deleted:
         return query
-
+    if org_check:
+        return query
```

Comment:
`org_check` isn't used anywhere?

Reply:
I made up `org_check` specifically for this application